src/alias/azext_alias/alias.py

```diff
@@ -0,0 +1,322 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import re
+import hashlib
+import json
+from configparser import ConfigParser
```

Comment:
Would prefer you use `from six.moves import configparser`.
See https://pythonhosted.org/six/.

The configparser library you used hasn't been updated since 2016 and not sure you really need it either.

Reply:
Just remember why i didn't use `six.moves.configparser`: in python 2, `six.moves.configparser` is not able to detect duplicated sections and options in the config file. That means that users can create duplicated aliases or duplicated command section within the same alias without raising an error.

Reply:
What about this?
```
When strict is True (the default), the parser won’t allow for any section or option duplicates while reading from a single source (file, string or dictionary), raising DuplicateSectionError or DuplicateOptionError. 
https://docs.python.org/3/library/configparser.html#configparser.ConfigParser
```

It would only work on Python 3 since I couldn't find that same note on Python 2 but I don't think it's worth adding a 2 year old extension just for that.
If you have duplicate sections, then I assume just the last one would be registered. That seems ok to me.